packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketVerify/CoinmarketVerifyOptionsItem.tsx

```diff
@@ -56,6 +59,16 @@ export const CoinmarketVerifyOptionsItem = ({
             </Row>
         );
     }
+
+    if (!receiveNetwork) {
+        return null;
+    }
```

Comment:
Same as above in `CoinmarketVerifyOptions.tsx`

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/14294/commits/8835cb5273c4bb999ec1a3e73a4c86241fdfa730